wcfsetup/install/files/acp/update_com.woltlab.wcf_5.2.10_orphanedComments.php

```diff
@@ -0,0 +1,21 @@
+<?php
+namespace wcf\acp;
+
+use wcf\data\object\type\ObjectTypeCache;
+use wcf\system\WCF;
+
+// Earlier versions did remove moderation queues without taking care of
+// comments associated with them.
+$commentObjectTypeID = ObjectTypeCache::getInstance()->getObjectTypeIDByName(
+	"com.woltlab.wcf.comment.commentableContent",
+	"com.woltlab.wcf.moderation.queue"
+);
+
+$sql = "DELETE FROM     wcf" . WCF_N . "_comment
```

Comment:
Possibly also adjust this to use `CommentHandler`, but a bare query within the update script is acceptable to me.

Reply:
Looking into `CommentHandler` it does quite a lot of operations, so the update script should also use `CommentHandler`.